Notify/Notify/Notify/ViewModels/LoginPageViewModel.cs

```diff
@@ -8,17 +10,16 @@
 
 namespace Notify.ViewModels
 {
-    public class LoginPageViewModel : BaseViewModel
+    public class LoginPageViewModel : BaseViewModel, INotifyPropertyChanged
```

Comment:
What was this interface, and why don't we need it?

Reply:
I realized its implemented in `BaseViewModel` already